packages/graphql/src/context/BaseContext.ts

```diff
@@ -0,0 +1,24 @@
+import type { BaseActions } from '../actions/BaseActions'
+import { App, Wizard } from '../entities'
+import type { Project } from '../entities/Project'
```

Comment:
> but is there any real benefit in this case (since the Project` module does have some runtime exports)?

Yeah, it avoids the `require` in the transpiled JS. In this case it's such a minor cost that it doesn't really matter, but you can think of circumstances where you'd want to e.g. import a type from `@packages/server` without worrying about the entire code being imported executed since it potentially have side-effects. The `import type` is also important if we want to be able to remove the `nxs.gen` file from git since there's a sort of chicken-and-egg problem of doing a runtime import of a file that doesn't yet exist. With `import type` you can sidestep that 

> but assuming there is some way to lint or enforce this.

Yep, this is enforced via the `"importsNotUsedAsValues": "error"` which I added to the `graphql` and `launchpad` projects. It'll be a type-error if you're only using as a type but you don't `import type`